Tools/CISupport/ews-build/events.py

```diff
@@ -147,11 +197,19 @@ def buildFinished(self, key, build):
         builder = yield self.master.db.builders.getBuilder(build.get('builderid'))
         builder_display_name = builder.get('description')
 
+        patch_id = self.extractProperty(build, 'patch_id')
+
+        if self.extractProperty(build, 'github.number'):
+            return self.buildFinishedGitHub(builder_display_name, build)
+
+        if not patch_id:
```

Comment:
Do we even need this check? We didn't had it before. If we don't have it, it will try to send data to ews django app, and django app would reject it. I guess that might log an error in logs (i haven't checked though).

Reply:
We also didn't have tons of builds without `patch_id` before, though. Which is why I think we should have keep this check.